pkg/deploy-manager/subscription.go

```diff
@@ -126,6 +126,8 @@ func (t *DeployManager) generateClusterObjects(ocsCatalogImage string, subscript
 	ocsOG.SetGroupVersionKind(schema.GroupVersionKind{Group: operatorv1.SchemeGroupVersion.Group, Kind: "OperatorGroup", Version: operatorv1.SchemeGroupVersion.Version})
 
 	co.operatorGroups = append(co.operatorGroups, ocsOG)
+
+	// Catalog Sources
```

Comment:
```suggestion
	// Catalog Source
```

Reply:
Although just one Cat Src is there it's the name of the section so I think it should be plural only. Also if you see the other resources are in plural only, Like Namespaces, Subscriptions, Operator groups. The notion is to put how many of them might be in this place and append them all to the list.